docs/community-ee/community-ee-release-process.md

```diff
@@ -18,145 +18,63 @@ Versioning would be the core tag + a patch number, e.g
-
-14. Push both the images (general versioning and the latest) to Github Registry
-
-```
-podman push ghcr.io/ansible-community/community-ee-base:<ansible-core-version-base-ee-version>
-podman push ghcr.io/ansible-community/community-ee-base:latest
-```
-
-15. Commit and push the changes made in the `execution-environments/community-ee-base/execution-environment.yml` to the `https://github.com/ansible-community/images` repo.
-
-```
-git add execution-environments/community-ee-base/execution-environment.yml
-git commit
-git push origin <ansible-core-version-base-ee-version>
-```
-Compare and create the pull request.
-
-16.   Check the community-ee-base images [here](https://github.com/orgs/ansible-community/packages/container/package/community-ee-base)  and get the sha256 sum.
-
-
-
-## Build steps for Community-ee-minimal
-
-1. Go to the working directory
-
-`cd images/execution-environments/community-ee-minimal`
-
-2. Create a new release branch
-
-`git checkout -b base-<ansible-core-version-minimal-ee-version>`
-
-3. Edit the `execution-environment.yml` file to the right `collection version` and `ansible-core version`. Get the correct [ansible-core](https://pypi.org/project/ansible-core/) version and edit accordingly.
-
-`vim execution-environments/community-ee-minimal/execution-environment.yml`
-
-
-4. List images in local storage
-
-`podman images`
-
-5. Delete the existing community-ee-minimal image
-
-`podman rmi <podman_community-ee-minimal_IMAGE_ID>`
-
-6. Build the ansible community-ee-minimal podman image with ansible-builder. (versioning: 2.16.1-1)
-
-`ansible-builder build --tag ghcr.io/ansible-community/community-ee-minimal:<ansible-core-version-minimal-ee-version>`
-
-7. Check if the image has been created or not and get the <image ID> of `community-ee-minimal`
-
-`podman images`
-
-8. Build `latest` tag for the community-ee-minimal image
-
-`podman tag <image id> ghcr.io/ansible-community/community-ee-minimal:latest`
-
-9. Push both the images (general versioning and the latest) to Github Registry by using the Github Token created before
-
-```
-podman push ghcr.io/ansible-community/community-ee-minimal:<ansible-core-version-minimal-ee-version>
-podman push ghcr.io/ansible-community/community-ee-minimal:latest
-```
-10. Commit and push the changes made in the `execution-environments/community-ee-minimal/execution-environment.yml` to the `https://github.com/ansible-community/images` repo.
-
-```
-git add execution-environments/community-ee-minimal/execution-environment.yml
-git commit
-git push origin <ansible-core-version-minimal-ee-version>
-```
-Compare and create the pull request.
-
-11.   Check the community-ee-minimal images [here](https://github.com/orgs/ansible-community/packages/container/package/community-ee-minimal)  and get the sha256 sum.
+      - Go to the [ansible-community/images](https://github.com/ansible-community/images) repo
+      - Click on the ![`Actions`](action-workflow-block.png) to open the `Workflows`
```

Comment:
```suggestion
      - Click on the ![`Actions`](action-workflow-block.png) tab to view the available workflows.
```

Reply:
I'd recommend removing the screengrabs here. We don't really need to document a third-party UI.